### PR TITLE
Doc generation fix

### PR DESCRIPTION
**Pull request by Shreyas-vgr**
Doc generation fix

## Description
Fixed typedocs generation bugs and provided an `npm run docs` command to generate api_docs.
Also removed lerna gulp and travis.yml dependencies since we have migrated to GitHub actions.

## Testing
Generated the api_docs and verified all ts files being generated.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs(Add new document content)
- [ ] Translate Docs(Translate document content)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] I have read the **README** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
     "ask-sdk-dynamodb-persistence-adapter",
     "ask-sdk-runtime",
     "ask-sdk-s3-persistence-adapter",
-    "ask-sdk-v1adapter"
+    "ask-sdk-v1adapter",
+    "ask-sdk-express-adapter",
+    "ask-smapi-sdk"
```
Comment (ShenChen93) [inline lerna.json]: SMAPI SDK should not be included

File: package.json [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
-		"tsc": "npm run gulp -- tsc",
-		"tslint": "npm run gulp -- tslint",
-		"test": "npm run gulp -- test",
-		"clean-build": "npm run gulp -- clean",
-		"clean-node_modules": "npm run lerna -- clean",
-		"clean-all": "npm run clean-build && npm run clean-node_modules"
+		"setup": "./node_modules/.bin/lerna exec npm install",
+		"bootstrap": "./node_modules/.bin/lerna bootstrap --force-local --hoist",
+		"compile": "./node_modules/.bin/lerna exec npm run compile",
+		"docs": "rm -rf ./api_docs && typedoc --out api_docs"
```
Comment (ShenChen93) [inline package.json]: typedoc should also use the local one: `./node_modules/typedoc` ?

Comment (Shreyas-vgr) [reply package.json]: Ahhh, updated in latest commit.
Thanks :)

File: lerna.json (hunk 1 above)
Comment (Shreyas-vgr) [reply lerna.json]: Updated in latest commit :)

**Review by ShenChen93 — APPROVED** [review]

**Review by mikelid-zz — APPROVED** [review]

**Pull request merged**